File: 2-Rekursjon/Rekkesum.java
public class Rekkesum {

	public static int rekkesum(int n) {
  	if (n == 1) return 1;
  	return rekkesum(n-1) + n;
	} 

 public static void main(String[] args) {
    int n = Integer.parseInt(args[0]);
		System.out.println(" " +  n);
    System.out.println(" ∑i = " + rekkesum(n));
		System.out.println("i=1");
  }

}

File: 2-Rekursjon/rekkesum.c
#include <stdio.h>
#include <stdlib.h>

int rekkesum(int n) {
  if (n == 1) return 1;
  return rekkesum(n-1) + n;
} 

int main(int argc, char **argv) {
  printf("%i\n", rekkesum(atoi(argv[1])));
	return 0;
}
